meta-xt-driver-domain/recipes-extended/camerabe/files/camera_be.cfg

```diff
@@ -19,7 +19,7 @@ mediactl:
 {
     link1 = "'rcar_csi2 feaa0000.csi2':1 -> 'VIN0 output':0 [1]";
     link2 = "'adv748x 4-0070 hdmi':1 -> 'adv748x 4-0070 txa':0 [1]";
-    source_fmt = "'adv748x 4-0070 hdmi':1 [fmt:RGB888_1X24/1920x1080 field:none]";
+    source_fmt = "'adv748x 4-0070 hdmi':1 [fmt:RGB888_1X24/1920x1080 field:none],'adv748x 4-0070 txa':0 [fmt:RGB888_1X24/1920x1080 field:none]";
     sink_fmt = "'rcar_csi2 feaa0000.csi2':1 [fmt:RGB888_1X24/1920x1080 field:none]";
```

Comment:
I am wondering, doesn't it work without that change? Is this suggested elsewhere?